### PR TITLE
Adds 15.6 release notes

### PR DESCRIPTION
**Pull request by axosoft-ramint**

### CODE REVIEW

File: gitlens/gitlens-release-notes-current.md [hunk 1]
```diff
@@ -16,6 +16,84 @@ Features marked with `PREVIEW` require a GitKraken Account, with access level ba
 
 ---
 
+<a id="v15-6"></a>
+
+## Version 15.6
+
+#### Thursday, September 12, 2024
```
Comment (d13) [inline gitlens/gitlens-release-notes-current.md:23]: October 3,

File: gitlens/gitlens-release-notes-current.md [hunk 2]
```diff
@@ -16,6 +16,84 @@ Features marked with `PREVIEW` require a GitKraken Account, with access level ba
...
+
+GitLens 15.6 arrives with easier branch workflows from within the Commit Graph, Account View integrated into the Home View, improved Cursor support, and a host of other improvements and bug fixes.
+
+<img src="/wp-content/uploads/gl-15-6-hero.png" class="img-responsive center img-bordered">
+
+### Commit Graph Improvements
+
+GitLens 15.6 includes several new features and improvements for the Commit Graph:
+
+The graph toolbar now includes a new _Create New Branch_ button in the upper-right corner that allows you to conveniently create a new branch from the current branch or commit. Furthermore, when your current branch is both ahead and behind its upstream, we've added a new _Force Push_ button to the toolbar next to the _Pull_ button that allows you to force push your current branch to its upstream (always force push with caution, as you may lose commits). These new buttons allow for easier entry into common branch workflows directly from the Commit Graph.
```
Comment (d13) [inline gitlens/gitlens-release-notes-current.md:33]: > conveniently create a new branch from the current branch or commit.

Do we want to mention opening branch in a worktree?

File: gitlens/gitlens-release-notes-current.md (hunk 1 above)
Comment (axosoft-ramint) [reply gitlens/gitlens-release-notes-current.md]: Good catch, thanks!

File: gitlens/gitlens-release-notes-current.md (hunk 2 above)
Comment (axosoft-ramint) [reply gitlens/gitlens-release-notes-current.md:33]: Did we add functionality for that? I don't see it mentioned in the changelog and can't find a button for it in the graph

Comment (d13) [reply gitlens/gitlens-release-notes-current.md:33]: Its an option on the confirmation quickpick for creating the branch

Comment (axosoft-ramint) [reply gitlens/gitlens-release-notes-current.md]: Updated the wording

**Pull request merged**